PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,9 @@
 	3. Submit the PR.
 -->
 
-{Write down the list of resources or topics you have added.}
+{Computer Vision - Convolutional Neural Networks for Visual Recognition.}
 
-{If you have added a resource, please explain your experience with the resource in context of why you have added it.}
+{This is a course provided by Stanford.edu, In this tutorial students can get complete information of the course (Convolutional Neural Networks for Visual Recognition) along with notes, code snippets and a complete youtube playlist of the course.}
```

Comment:
@Prince-Shivaram I don't think you can change this. It's like a guide that you should follow this template while raising the PR. You don't need to change here.